release/preview/nanoserver1903/getLatestTag.ps1

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft Corporation. 
```

Comment:
Delete this file in all the preview nanoserver folders

Reply:
deleted